inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/metrics/prometheus/PrometheusMetricListener.java

```diff
@@ -40,17 +56,28 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.prometheus.client.Collector;
+import io.prometheus.client.CounterMetricFamily;
+import io.prometheus.client.exporter.HTTPServer;
+
 /**
  * 
  * PrometheusMetricListener
  */
-public class PrometheusMetricListener implements MetricListener {
+public class PrometheusMetricListener extends Collector implements MetricListener {
 
-    public static final Logger LOG = LoggerFactory.getLogger(MetricRegister.class);
+    private static final Logger LOG = LoggerFactory.getLogger(MetricRegister.class);
```

Comment:
Whether this logger should get from `PrometheusMetricListener.class`?
```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(PrometheusMetricListener.class);
```